llvm/test/SYCLLowerIR/pfwg_and_pfwi.ll

```diff
@@ -70,4 +74,15 @@ bb:
   ret void
 }
 
+define internal spir_func void @foo(ptr addrspace(4) %arg, ptr byval(%struct.foo.0) align 1 %arg1) align 2 !work_group_scope !0 {
+bb:
+  call spir_func void @baz(ptr addrspace(4) %arg, ptr byval(%struct.foo.0) align 1 %arg1)
```

Comment:
I wonder what happens if there are additional instructions in foo (in work group scope) except parallel_for_work_item call. Is this handled properly or is it a separate bug?